_includes/cookie-notice.html

```diff
@@ -0,0 +1,118 @@
+<!-- Cookies consent -->
+<style>
+  #cookie-notice {
+    display: none;
+    line-height: 40px;
+    position: fixed;
+    justify-content: center;
+    align-items: center;
+    gap: 60px;
+    z-index: 999;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    padding: 16px 20px;
+    text-align: center;
+    border: 1px solid var(--card-border);
+    background-color: var(--input-search-background);
+    color: var(--body-text);
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 22px;
+  }
+  #cookie-notice p {
+    margin: 0;
+    text-align: left;
+  }
+
+  #cookie-notice a.btn {
+    min-width: 26px;
+    margin-right: 5px;
+    cursor: pointer;
+    text-align: center;
+    white-space: nowrap;
+    outline: none;
+    padding: 12px 40px;
+    gap: 2px;
+    border: 1px solid var(--body-text);
+    border-radius: 24px;
+    font-family: "Space Grotesk", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 24px;
+    text-transform: capitalize;
+    color: var(--body-text);
+    min-width: 148px;
+    height: 48px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  #cookie-notice a.btn:hover {
+    background-color: var(--post-link-hover);
+    color: var(--button-text);
+    border: 1px solid var(--post-link-hover);
+  }
+  @media (max-width: 767px) {
+    #cookie-notice {
+      gap: 20px;
+      flex-wrap: wrap;
+    }
+  }
+</style>
+
+<div id="cookie-notice">
+  <p>
+    This website uses cookies to ensure you get the best browsing experience. By
+    continuing to use this website, you consent to our use of these cookies.
+    This website contains proprietary content that belongs to LivePerson and
+    that is intended for educational use by our customers and prospects. Your
+    use of this site is subject to our Terms of Use, which include restrictions
```

Comment:
I know it’s missing on the screen but please link “Terms of Use” (→ https://www.liveperson.com/policies/terms-of-use/).